tests/all-tests.html

```diff
@@ -72,13 +52,7 @@
         "./component-tests/tabs/html/Tabs-test.html",
         "./component-tests/tooltip/html/Tooltip-test.html",
         "./component-tests/uiOptions/html/UIOptions-test.html",
-        "./component-tests/uploader/html/DemoUploadManager-test.html",
-        "./component-tests/uploader/html/ErrorPanel-test.html",
-        "./component-tests/uploader/html/FileQueue-test.html",
-        "./component-tests/uploader/html/FileQueueView-test.html",
-        "./component-tests/uploader/html/HTML5UploaderSupport-test.html",
-        "./component-tests/uploader/html/Uploader-test.html",
-        "./component-tests/uploader/html/UploaderCompatibility-test.html",
+        "./component-tests/uploader/all-tests.html",
```

Comment:
Does this work in all browsers. I feel like I had issues before with the nesting of iframes in iframes.